### ISSUE TITLE
Suggestions, security related

### ISSUE THREAD

**Issue by eurobank** [CIDRAM/CIDRAM#216]
After recently installing a website server with CIDRAM, i would like to make two suggestions.

1. Please remove the "Default user Admin and Default password password" text from login page. No reason to have that. 
2. Please think about NOT providing so much and detailed information in a Block page (via the default template). No reason to inform the visitor about script used, signatures etc. Why give so many detaile? A reason for bocking would be enough (and more than enough) i think. 

thanks

**Comment by Maikuolan** [comment]
> 1. Please remove the "Default user Admin and Default password password" text from login page. No reason to have that.

That's mostly just an aesthetic element of the front-end anyway, so it shouldn't be a problem to remove it. I have been considering overhauling many of themes recently anyway, so I could always just consider removing that text as part of the overhaul, getting rid of it as well as whatever other changes.

> 2. Please think about NOT providing so much and detailed information in a Block page (via the default template). No reason to inform the visitor about script used, signatures etc. Why give so many detaile? A reason for bocking would be enough (and more than enough) i think.

Planning on adding some nice new options for that with v3, so that'll definitely happen. :-)

(There are a few details to sort out first, but I plan to start working on v3 at some point within the next few months, maybe earlier, depending on how long it takes to sort out those other details first).

The reason for not doing it earlier (v1, v2), is that there are some existing configuration options, which the new ones will supersede and completely replace, and I aim to treat the removal of existing configuration options as a "backwards-incompatible change", warranting a new major version release. But anyway, it shouldn't be too far in the future.

**Comment by Maikuolan** [comment]
The first of the two suggestions has now been implemented. :-)

**Comment by 737simpilot** [comment]
Just want to add here that if you use the Obscured theme that'll strip out the type of script you're using, i.e. CIDRAM in the block page for the user.

**Comment by Maikuolan** [comment]
And.. For v3 at least, the second of the two suggestions has now been implemented (2ac0100e0ba789d946269fc0e38a75bb70deb45e). :-)

I'm thinking I might try doing something to make the obscured theme redundant, too (i.e., so that the core can cover that functionality without the need for the theme). I'll keep this issue open for a few more days as a self-reminder.

**Comment by Maikuolan** [comment]
> I'm thinking I might try doing something to make the obscured theme redundant, too (i.e., so that the core can cover that functionality without the need for the theme). I'll keep this issue open for a few more days as a self-reminder.

And done. :-)

*Changelog excerpt*:
```
- [2022.03.10; NEW FEATURE; Maikuolan]: Added two new configuration directives,
  "block_event_title" and "captcha_title", allowing users to customise the page
  title used for block events and CAPTCHA requests (#216).
```
